CHANGELOG.md

```diff
@@ -2,6 +2,96 @@
 
 This document outlines major changes between releases.
 
+## 0.107.0 "Multithreading/Multiplication" (02 Dec 2024)
```

Comment:
Voting is open, suggestions are welcome. Multiplication is boring, Multithreading doesn't fit the naming policy.

Reply:
mongrelization